plugins/default/manifest.json

```diff
@@ -421,6 +421,27 @@
         },
         "required": ["format"]
       }
+    },
+    {
```

Comment:
Hey. Can you please follow the same structure as used by other plugins in this file?